src/response-factory.js

```diff
@@ -22,7 +22,7 @@ exports.createStepNameResponse = function (message, messageId) {
       stepName: [],
       hasAlias: false,
       fileName: "",
-      lineNumber: -1
+      span: null
```

Comment:
pass empty object `{}` instead of `null` as the default value. It can cause `TypeError: Cannot read property 'start' of null`.